src/test/java/com/xiaomi/infra/pegasus/client/TestBasic.java

```diff
@@ -2632,87 +2631,27 @@ private void testWriteSizeLimit(PegasusClientInterface client) {
-    // case B2: scan limit record by "expired_0" and "", if persistent record count < maxFetchCount,
-    // it only return valid records
-    ScanRangeResult caseB2 =
-        table.scanRange(
-            hashKey.getBytes(), "expired_0".getBytes(), "".getBytes(), new ScanOptions(), 50, 0);
-    assertScanResult(0, 9, true, caseB2);
-    // case B3: scan limit record by "persistent_5" and "", if following persistent record count <
-    // maxFetchCount, it only return valid records
-    ScanRangeResult caseB3 =
-        table.scanRange(
-            hashKey.getBytes(), "persistent_5".getBytes(), "".getBytes(), new ScanOptions(), 50, 0);
-    assertScanResult(5, 9, true, caseB3);
-    // case B4: scan limit record by "persistent_5" and "", if following persistent record count >
-    // maxFetchCount, it only return valid records
-    ScanRangeResult caseB4 =
-        table.scanRange(
-            hashKey.getBytes(), "persistent_5".getBytes(), "".getBytes(), new ScanOptions(), 3, 0);
-    assertScanResult(5, 7, false, caseB4);
-
-    // case C: scan limit record by "" and "stopSortKey":
-    // case C1: scan limit record by "" and "expired_7", if will return 0 record
-    ScanRangeResult caseC1 =
-        table.scanRange(
-            hashKey.getBytes(), "".getBytes(), "expired_7".getBytes(), new ScanOptions(), 3, 0);
-    Assert.assertTrue(caseC1.allFetched);
-    Assert.assertEquals(0, caseC1.results.size()); // among "" and "expired_7" has 0 valid record
-    // case C2: scan limit record by "" and "persistent_7", if valid record count < maxFetchCount,
-    // it only return valid record
-    ScanRangeResult caseC2 =
-        table.scanRange(
-            hashKey.getBytes(), "".getBytes(), "persistent_7".getBytes(), new ScanOptions(), 10, 0);
-    assertScanResult(0, 6, true, caseC2);
-    // case C3: scan limit record by "" and "persistent_7", if valid record count > maxFetchCount,
-    // it only return valid record
-    ScanRangeResult caseC3 =
-        table.scanRange(
-            hashKey.getBytes(), "".getBytes(), "persistent_7".getBytes(), new ScanOptions(), 2, 0);
-    assertScanResult(0, 1, false, caseC3);
-
-    // case D: use multiGetSortKeys, which actually equal with case A but no value
-    // case D1: maxFetchCount > 0, return maxFetchCount valid record
-    MultiGetSortKeysResult caseD1 = table.multiGetSortKeys(hashKey.getBytes(), 5, -1, 0);
-    Assert.assertFalse(caseD1.allFetched);
-    Assert.assertEquals(5, caseD1.keys.size());
+    MultiGetSortKeysResult result1 = table.multiGetSortKeys(hashKey.getBytes(), 5, -1, 0);
+    Assertions.assertFalse(result1.allFetched);
+    Assertions.assertEquals(5, result1.keys.size());
     for (int i = 0; i <= 4; i++) {
-      Assertions.assertEquals("persistent_" + i, new String(caseD1.keys.get(i)));
+      Assertions.assertEquals("persistent_" + i, new String(result1.keys.get(i)));
     }
     // case D1: maxFetchCount < 0, return all valid record
```

Comment:
case A,B,C have been moved to TestRange.java, how about update comment for `case D1`?